app/controllers/dinosaurs_controller.rb

```diff
@@ -7,6 +7,15 @@ def index
 
   def show
     @dinosaur = Dinosaur.find(params[:id])
+    @review = Review.new
+  end
+
+  def destroy
```

Comment:
There are now two destroy methods in this controller. I'd sync up with the other half of the team and figure out how to merge what you each did into a single method.